test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/NetCorePackageReferenceProjectTests.cs

```diff
@@ -0,0 +1,483 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+using NuGet.VisualStudio;
+using NuGet.Test.Utility;
+using System.Threading;
+using System.IO;
+using Moq;
+using FluentAssertions;
+using NuGet.ProjectModel;
+using NuGet.Commands.Test;
+using NuGet.Commands;
+using NuGet.Configuration;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+
+namespace NuGet.PackageManagement.VisualStudio.Test
+{
+    public class NetCorePackageReferenceProjectTests
+    {
+        [Fact]
+        public async Task GetInstalledVersion_WithAssetsFile_ReturnsVersionsFromAssetsSpecs()
```

Comment:
I'm not sure this test verifies that something is actually being read from the assets file. 

The same version is specified in the package spec and the assets file :)